news/tests/test_tasks.py

```diff
@@ -19,9 +19,36 @@
     send_message,
     SUBSCRIBE,
     update_user,
+    get_lock,
+    RetryTask,
 )
 
 
+@override_settings(TASK_LOCKING_ENABLE=True)
+class TaskDuplicationLockingTests(TestCase):
+    def test_locks_work(self):
+        """Calling get_lock more than once quickly with the same key should be locked"""
+        get_lock('dude@example.com')
+        with self.assertRaises(RetryTask):
+            get_lock('dude@example.com')
+
+    def test_lock_prefix_works(self):
+        """Should allow same key to not lock other prefixes"""
+        get_lock('dude@example.com', prefix='malibu')
```

Comment:
Heh. Maybe? I was just thinking of places The Dude went. Malibu is where the party at Jackie Treehorn's place was, and who's sheriff threw a coffee mug at him 🍸